### DISCUSSION TITLE
Problème pour ajouter les vannes aux thermostats

### DISCUSSION THREAD

**Discussion by HervePOG** [jmcollin78/versatile_thermostat#644] [Q&A] [answered]
Bonjour à tous,
J'ai installé VERSATILE THERMOSTAT sur mon HA dans le but de gérer plus finement des vannes thermostatiques.
L'outils me semble très complet et bien adapaté à mes besoins mais je rencontre un problème pour associer mes vannes aux thermostats.
Mes vannes sont connectées via Zigbee2MQTT et fonctionnent normalement depuis des mois :

> ![image](https://github.com/user-attachments/assets/13d80727-96db-4940-a4b9-ecf5e3ff7c2a)

A la création du thermostat, cela commence ainsi :  

> ![image](https://github.com/user-attachments/assets/14ad1fef-7166-4348-ad43-51fb5ad620d3)

Puis il faut choisir l'élément à piloter et je ne trouve que des calibrations et non la vanne elle-même : 

> ![image](https://github.com/user-attachments/assets/3b671b9f-755a-431e-b438-9f1aba79c9da)

J'ai beau chercher dans la documentions, depuis des jours, je séche ! Même ChatGPT n'a rien trouver.
Merci à celui ou ceux qui pourront me donner des conseils.

Hervé

**Comment by jmcollin78** [comment, marked as answer]
Hello @HervePOG ,

Si t'as vanne n'a pas d'entité number qui permet de piloter l'ouverture de celle-ci, tu ne peux pas utiliser le Vtherm `over_valve` mais tu dois utiliser le `over_climate`. C'est moins bien mais j'ai pas mieux à te proposer.

C'est écrit ici pour info: https://github.com/jmcollin78/versatile_thermostat/blob/main/README-fr.md#s%C3%A9lectionnez-des-entit%C3%A9s-pilot%C3%A9es-sous-jacents

**Reply by HervePOG** [reply]
Merci beaucoup pour ta réponse si rapide, et qui vaut bien une bière !
Cela solutionne en effet mon problème. Effectivement c'est un peu moins bien avec over-climate mais cela devrait fonctionner, je teste...
Il faudra que je fasse attention pour choisir les prochaines vannes, mais c'est probablement difficile de savoir avant d'essayer.

Merci encore !

Hervé

**Reply by jmcollin78** [reply]
Merci pour la bière. A la tienne ! 🍻
